### PR TITLE
Add issue template for event hubs

### PR DESCRIPTION
**Pull request by choldgraf**
This adds an issue template for event-based hubs. This is part of #790 , where we could have avoided some issues by having a set list of tasks to follow before the hub, to make sure things were behaving as expected.

### Feedback

I'd love feedback about whether the information that's in there is the "right" information to ask for, and whether the tasks look correct to folks.

### CODE REVIEW

**Review by damianavila — APPROVED** [review]
This feels like the right content to me as a starting point.

File: .github/ISSUE_TEMPLATE/5_event-hub.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+### Task List
+
+- [ ] Dates confirmed with the community representative
+- [ ] **One week before event** Hub is running.
+- [ ] Confirm with Community Representative that their workflows function as expected.
+  - Log-in and authentication works as-expected
+  - Any `nbgitpuller` links resolve properly
+  - Notebooks run as-expected
+- [ ] **1 day before event** Hub nodes are pre-cooked.
```
Comment (yuvipanda) [inline .github/ISSUE_TEMPLATE/5_event-hub.md:30]:
```suggestion
- [ ] **1 day before event** A separate nodegroup is provisioned for the event

```

File: .github/ISSUE_TEMPLATE/5_event-hub.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- [ ] Event in progress.
+- [ ] Confirm event is finished.
+- [ ] Hub pre-cooked nodes are wound down.
```
Comment (yuvipanda) [inline .github/ISSUE_TEMPLATE/5_event-hub.md:33]:
```suggestion
- [ ] Nodegroup created for the hub is decommissioned.

```

**Review by yuvipanda — APPROVED** [review]

**Comment by choldgraf** [comment]
thanks all!

**Pull request merged**